controllers/teams.js

```diff
@@ -572,20 +572,14 @@ router.get('/:teamId', async (req, res, next) => {
 			.slice(0, 6);
 
 		const news = await api(req)
-			.get('/news/', {
+			.get(`/v3/news/team/${req.params.teamId}`, {
```

Comment:
I would prefer the team id to be a query param. (https://github.com/hpi-schul-cloud/schulcloud-server/pull/2673/files#r659948565)